src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Routes from './router';
+
+
+
+ReactDOM.render(<Routes />, document.getElementById('index'));
```

Comment:
Cool stuff ! What do you feel about using object destructuring here? like `import { render } from 'react-dom';` instead of importing the whole module.

Reply:
I actually did not create this file, but I like the idea, I will look into it.  Thanks for spotting it out.